consensus/hotstuff/forks/forks2_test.go

```diff
@@ -311,148 +312,148 @@ func TestDuplication(t *testing.T) {
+		// adding block [◄(1) 5]: note that QC is _below_ the pruning threshold, i.e. cannot resolve the parent
 		// * Forks should store block, despite the parent already being pruned
 		// * finalization should not change
-		certBlockWithUnknownParent := toCertifiedBlock(t, blocks[3].Block)
+		certBlockWithUnknownParent := toCertifiedBlock(t, blocks[3])
 		require.Nil(t, forks.AddCertifiedBlock(certBlockWithUnknownParent))
 		require.True(t, forks.IsKnownBlock(certBlockWithUnknownParent.Block.BlockID))
-		requireLatestFinalizedBlock(t, forks, blocks[0].Block)
+		requireLatestFinalizedBlock(t, forks, blocks[0])
 		requireFinalityProof(t, forks, expectedFinalityProof)
 	})
 }
 
 // TestDoubleProposal tests that the DoubleProposal notification is emitted when two different
-// proposals for the same view are added. We ingest the the following block tree:
+// blocks for the same view are added. We ingest the the following block tree:
 //
-//	               / [(◄1) 2]
+//	               / [◄(1) 2]
 //			[1]
-//	               \ [(◄1) 2']
+//	               \ [◄(1) 2']
 //
-// which should result in a DoubleProposal event referencing the blocks [(◄1) 2] and [(◄1) 2']
+// which should result in a DoubleProposal event referencing the blocks [◄(1) 2] and [◄(1) 2']
 func TestDoubleProposal(t *testing.T) {
 	blocks, err := NewBlockBuilder().
-		Add(1, 2).                // [(◄1) 2]
-		AddVersioned(1, 2, 0, 1). // [(◄1) 2']
+		Add(1, 2).                // [◄(1) 2]
+		AddVersioned(1, 2, 0, 1). // [◄(1) 2']
 		Blocks()
 	require.Nil(t, err)
 
-	t.Run("ingest proposals", func(t *testing.T) {
+	t.Run("consensus participant mode: ingest validated blocks", func(t *testing.T) {
 		forks, notifier := newForks(t)
-		notifier.On("OnDoubleProposeDetected", blocks[1].Block, blocks[0].Block).Once()
+		notifier.On("OnDoubleProposeDetected", blocks[1], blocks[0]).Once()
 
-		err = addProposalsToForks(forks, blocks)
+		err = addValidatedBlockToForks(forks, blocks)
 		require.Nil(t, err)
 	})
 
-	t.Run("ingest certified blocks", func(t *testing.T) {
+	t.Run("consensus follower mode: ingest certified blocks", func(t *testing.T) {
 		forks, notifier := newForks(t)
-		notifier.On("OnDoubleProposeDetected", blocks[1].Block, blocks[0].Block).Once()
+		notifier.On("OnDoubleProposeDetected", blocks[1], blocks[0]).Once()
 
-		err = forks.AddCertifiedBlock(toCertifiedBlock(t, blocks[0].Block)) // add [(◄1) 2]  as certified block
+		err = forks.AddCertifiedBlock(toCertifiedBlock(t, blocks[0])) // add [◄(1) 2]  as certified block
 		require.Nil(t, err)
-		err = forks.AddCertifiedBlock(toCertifiedBlock(t, blocks[1].Block)) // add [(◄1) 2']  as certified block
+		err = forks.AddCertifiedBlock(toCertifiedBlock(t, blocks[1])) // add [◄(1) 2']  as certified block
 		require.Nil(t, err)
 	})
 }
 
 // TestConflictingQCs checks that adding 2 conflicting QCs should return model.ByzantineThresholdExceededError
 // We ingest the the following block tree:
```

Comment:
```suggestion
// We ingest the following block tree:
```

Reply:
applied in https://github.com/onflow/flow-go/pull/4234/commits/3fa1776152863588863cd7d6c98ab137de5d2e3f